docs/joss_paper/paper.md

```diff
@@ -55,67 +67,67 @@ bibliography: paper.bib
 
-`DeepReg` provides command-line tool options that implement basic functionalities including model training, prediction and image warping. The yaml files and command-line arguments are used as the basis of configuring the various options with `DeepReg`. This design choice has been motivated by both research and education purposes originally set out in the `DeepReg` project, which aims to provide benchmarking implementations of standard deep-learning-based image registration algorithms. These implementations, together with their documentation, tutorials and demos, simplify workflows for prototyping and developing novel methodology, rapid utilising latest development in this field and accessing quality research topics. A MICCAI Educational Challenge has utilised the `DeepReg` code and demos to explore the link between classical algorithms and deep-learning-based methods, while a latest research paper investigated temporal changes for prostate cancer patients under active surveillance programme by using longitudinal image registration, adapted from the `DeepReg` code [@yang2020longitudinal].
+`DeepReg` provides command-line tool options that enabling the basic functionalities including model training, prediction and image warping. The yaml files and command-line arguments are used as the basis of configuring the various options with `DeepReg`. This design choice has been motivated by both research and education purposes originally set out in the `DeepReg` project, which aims to provide benchmarking implementations of standard deep-learning-based image registration algorithms. These implementations, together with their documentation, tutorials and demos, simplify workflows for prototyping and developing novel methodology, rapid utilising latest development in this field and accessing quality research topics. A MICCAI Educational Challenge has utilised the `DeepReg` code and demos to explore the link between classical algorithms and deep-learning-based methods, while a latest research paper investigated temporal changes for prostate cancer patients under active surveillance programme, by using longitudinal image registration adapted from the `DeepReg` code [@yang2020longitudinal].
```

Comment:
"while a latest research paper" maybe change to a research paper ? grammarly is complaining that it should be "the latest"